website/src/partials/_install.mdx

```diff
@@ -47,7 +47,7 @@ import TabItem from '@theme/TabItem';
 
   <TabItem value="win" label="Windows">
     {/* x-release-please-start-version */}
-    Download the [windows archive](https://github.com/glasskube/glasskube/releases/download/v0.0.1/glasskube_v0.0.1_windows_x86_64.zip) from our
+    Download the [windows archive](https://github.com/glasskube/glasskube/releases/download/v0.0.2/glasskube_v0.0.1_windows_x86_64.zip) from our
```

Comment:
@kosmoz release please seems to miss the version in the filename.

Reply:
https://github.com/googleapis/release-please/issues/2137

Reply:
solved with #228